# Cable
#port: /dev/ttyACM0
#baud: 57600
#rate: 20

# XBee
port: /dev/ttyUSB0
baud: 57600
rate: 10

timeout: 0.01
sensorstate_rate: 10

use_base_controller: True
base_controller_rate: 10
#base_frame: base_link

wheel_diameter: 0.146
wheel_track: 0.297
#encoder_resolution: 8384  # (from Pololu for 131:1 motors)
#encoder_resolution: 8100  # Empirical after calibration
encoder_resolution:  4331   # (from Pololu for 67.7:1 motors)
gear_reduction: 1.0
motors_reversed: True
Kp: 20
Kd: 12
Ki: 1
Ko: 50
accel_limit: 0.05

# List the sensors connected to the controller.  Examples only - edit for your robot.
sensors: {
  ir_front_right:       {pin: 15, type: GP2D12, rate: 5},
  ir_front_center:      {pin: 10, type: GP2D12, rate: 5},
  ir_front_left:        {pin: 9, type: GP2D12, rate: 5},
  sonar_front_right:    {pin: 46, type: Ping, rate: 5},
  sonar_front_center:   {pin: 47, type: Ping, rate: 5},
  sonar_front_left:     {pin: 48, type: Ping, rate: 5},
  motor_current_left:   {pin: 0, type: PololuMotorCurrent, rate: 2},
  motor_current_right:  {pin: 1, type: PololuMotorCurrent, rate: 2}
  #arduino_led:        {pin: 13, type: Digital, rate: 5, direction: output}
  #button:              {pin: 4, type: Digital, rate: 10, direction: input}
  #laser:               {pin: 5, type: Digital, rate: 2, direction: output}
  #sonar_back_right:     {pin: 44, type: Ping, rate: 5},
  #sonar_back_left:      {pin: 40, type: Ping, rate: 5},
}
